src/components/Announcement/index.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import * as React from 'react'
+import { useModal } from '@faceless-ui/modal'
+import { RichText } from '@components/RichText'
+import { ArrowIcon } from '@root/icons/ArrowIcon'
+import { CloseIcon } from '@root/graphics/CloseIcon'
+import { useCookies } from 'react-cookie'
+
+import classes from './index.module.scss'
+
+export const Announcement: React.FC<{ message?: String }> = ({ message }) => {
+  const { oneModalIsOpen } = useModal()
```

Comment:
I'm not sure we need to tie into modal context here, I see you're conditionally rendering based on if one is open, but it feels like this should always render and z-index on top or beneath the modal as needed.